tests/Unit/Helpers/ParallelAlgorithms/LinearSolver/LinearSolverAlgorithmTestHelpers.hpp

```diff
@@ -175,22 +176,11 @@ struct InitializeElement {
                     const Parallel::ConstGlobalCache<Metavariables>& cache,
                     const int /*array_index*/, const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    const auto& linear_operator = get<LinearOperator>(cache);
-    const auto& source = get<Source>(cache);
-    const auto& initial_guess = get<InitialGuess>(cache);
-
     return std::make_tuple(
         ::Initialization::merge_into_databox<
             InitializeElement,
-            db::AddSimpleTags<VectorTag, ::Tags::FixedSource<VectorTag>,
-                              LinearSolver::Tags::OperatorAppliedTo<VectorTag>,
-                              operand_tag, operator_tag>>(
-            std::move(box), initial_guess, source,
-            DenseVector<double>{linear_operator * initial_guess},
-            make_with_value<DenseVector<double>>(
-                initial_guess, std::numeric_limits<double>::signaling_NaN()),
-            make_with_value<DenseVector<double>>(
-                initial_guess, std::numeric_limits<double>::signaling_NaN())));
+            db::AddSimpleTags<VectorTag, ::Tags::FixedSource<VectorTag>>>(
+            std::move(box), get<InitialGuess>(cache), get<Source>(cache)));
```

Comment:
should be able to get these from the box

Reply:
It was easier to carry this change through both test helpers and push as separate commit